tour_model_eval/viz_similarity_unlabeled.ipynb

```diff
@@ -151,19 +153,57 @@
    "cell_type": "code",
    "execution_count": null,
    "id": "great-leeds",
-   "metadata": {},
+   "metadata": {
+    "scrolled": true
+   },
    "outputs": [],
    "source": [
-    "graph(sim)"
+    "graph(sim)\n"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "preceding-executive",
+   "id": "qualified-myrtle",
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "fig=plt.figure(1)\n",
+    "n = len(long_term_uuid_list)\n",
+    "for i in range(n):\n",
+    "    plt.subplot(n,n,i+1)\n",
+    "    data = eamtc.read_data(long_term_uuid_list[i])\n",
+    "    radius = 300\n",
+    "    sim = similarity.similarity(data, radius)\n",
+    "    sim.bin_data()\n",
+    "    sim.calc_cutoff_bins()\n",
+    "    \n",
+    "    graph(sim)\n",
+    "fig"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "infinite-pantyhose",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# #second sulotion\n",
```

Comment:
are you going to keep both solutions?
does this solution work? don't you need to pass in `ax` to the graph?

Reply:
I delete this solution. It seems work, but I don't know why I don't need to pass `ax` to the graph.

Reply:
I guess `plt.bar` uses the last subplot even if you don't pass in the `ax`